climpred/metrics.py

```diff
@@ -329,5 +410,29 @@ def _uacc(forecast, reference, control, running=None, reference_period=None):
     Returns:
         uacc_skill (xarray object): skill of uACC
     """
-    return np.sqrt(
-        _ppp(forecast, reference, control, running, reference_period))
+    return _ppp(forecast, reference, dim=dim, comparison=comparison)**.5
+
+
+ALL_HINDCAST_METRICS_DICT = {
```

Comment:
maybe this dict should replace `_get_metric/comparison_function`?

Reply:
Yeah I think a dictionary approach is fine if it makes sense and is clean.